### DISCUSSION TITLE
bug: run failed on IOS

### DISCUSSION THREAD

**Discussion by areeb-obenan** [ionic-team/capacitor#6638] [Q&A] [answered]
## Bug Report
I have created a simple ionic boilerplate app, It should run on ios but it is not running on my end, I created an issue on ionic repository (here is link for that https://github.com/ionic-team/ionic-framework/issues/27587#issue-1737452911 ). They said this my code is running fine on their end. How can i debug on my end? why i am unable to run on ios. you can clone code using (git clone https://github.com/areeb-gsoftconsulting/ionicapp.git). 
If this code runs on your end, please guide how can i run on IOS, i have shared my system details below as well.

### Capacitor Version
ionicapp git:(master) ✗ npx cap doctor
💊   Capacitor Doctor  💊 

Latest Dependencies:

  @capacitor/cli: 5.0.4
  @capacitor/core: 5.0.4
  @capacitor/android: 5.0.4
  @capacitor/ios: 5.0.4

Installed Dependencies:

  @capacitor/android: not installed
  @capacitor/cli: 5.0.4
  @capacitor/core: 5.0.4
  @capacitor/ios: 5.0.4

[success] iOS looking great! 👌

### Code Reproduction
you can clone my code by running following command
git clone https://github.com/areeb-gsoftconsulting/ionicapp.git

`npm --version` output: 9.6.6

`node --version` output: v20.2.0

`pod --version` output (iOS issues only):1.11.3

Ionic:

   Ionic CLI       : 7.1.1 (/usr/local/lib/node_modules/@ionic/cli)
   Ionic Framework : @ionic/react 7.0.10

Capacitor:

   Capacitor CLI      : 5.0.4
   @capacitor/android : not installed
   @capacitor/core    : 5.0.4
   @capacitor/ios     : 5.0.4

Utility:

   cordova-res : not installed globally
   native-run  : 1.7.2

System:

   NodeJS : v20.2.0 (/usr/local/Cellar/node/20.2.0/bin/node)
   npm    : 9.6.6
   OS     : macOS Monterey

**Comment by jcesarmobile** [comment, marked as answer]
It's also working fine on my end. Converting the issue to discussion.

What's your Xcode version? Capacitor 5 requires Xcode 14.1 or newer.
If using Xcode 14.3, then CocoaPods version should be 1.12.1 (still recommended to use that even if your Xcode is older)

**Reply by areeb-obenan** [reply]
I am using Xcode 13.2.1, Let me update xcode.

**Comment by areeb-obenan** [comment]
I just updated my Xcode to 14.2 and Project successfully run.
